### PR TITLE
First attempt at DeploySentinel and fix couchdb notebook tests

### LINKED ISSUE #6399: [e2e] Add deploysentinel to CI to aggregate test results
#### Summary
Deploysentinel provides a playwright test aggregate dashboard to capture historical runs with test metadata to reduce triage time.

- [x] Integrate Deploysentinel into github actions
- [x] Integrate Deploysentinel into circle ci
- [x] Integrate Deploysentinel into github PR comments

### PR DESCRIPTION
**Pull request by unlikelyzero**
WAIT FOR https://github.com/nasa/openmct/pull/6525

Closes #6399

### Describe your changes:
Adds deploysentinel to our CI configs
a quick driveby for cleaning up test files from gitignore
driveby documentation
driveby stability fixed for couchdb tests

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +85,7 @@
     "build:watch": "webpack --config ./.webpack/webpack.dev.js --watch",
     "info": "npx envinfo --system --browsers --npmPackages --binaries --languages --markdown",
     "test": "karma start",
+    "test:clean": "rm -rf ./coverage ./html-test-results ./test-results ./nyc_output/",
```
Comment (unlikelyzero) [inline package.json:88]: driveby

File: e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js [hunk 2]
```diff
@@ -275,3 +206,40 @@ function filterNonFetchRequests(requests) {
...
+ * @param {string} tagName
+ */
+async function removeTagAndAwaitNetwork(page, tagName) {
+    await page.hover(`[aria-label="Tag"]:has-text("${tagName}")`);
+    await Promise.all([
+        page.locator(`[aria-label="Remove tag ${tagName}"]`).click(),
+        //With this pattern, we're awaiting the response but asserting on the request payload.
+        page.waitForResponse(resp => resp.request().postData().includes(`"_deleted":true`) && resp.status() === 201)
+    ]);
+    await page.waitForSelector(`[aria-label="Tag"]:has-text("${tagName}")`, {state: 'hidden'});
```
Comment (ozyx) [inline e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js:243]: should we make this an assertion instead?
```js
expect(page.locator(`[aria-label="Tag"]:has-text("${tagName}")`).toBeHidden();
```

File: e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js [hunk 3]
```diff
@@ -275,3 +206,40 @@ function filterNonFetchRequests(requests) {
...
+async function addTagAndAwaitNetwork(page, tagName) {
+    await page.hover(`button:has-text("Add Tag")`);
+    await page.locator(`button:has-text("Add Tag")`).click();
+    await page.locator('[placeholder="Type to select tag"]').click();
+    await Promise.all([
+        // Waits for the next request with the specified url
+        page.waitForRequest('**/openmct/_all_docs?include_docs=true'),
+        // Triggers the request
+        page.locator(`[aria-label="Autocomplete Options"] >> text=${tagName}`).click(),
+        page.waitForSelector(`[aria-label="Tag"]:has-text('${tagName}')`)
```
Comment (ozyx) [inline e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js:225]: Convert to assertion also?

```js
expect(page.locator(`[aria-label="Tag"]:has-text("${tagName}")`).toBeVisible();
```

**Review by ozyx — CHANGES_REQUESTED** [review]
Looks good overall! Just have a few small suggestions

File: package.json (hunk 1 above)
Comment (ozyx) [reply package.json:88]: should be `./.nyc_output` ?

**Review by ozyx — APPROVED** [review]
LGTM

**Review by scottbell — APPROVED** [review]

**Pull request merged**